.travis.yml

```diff
@@ -20,20 +20,20 @@ matrix:
   include:
     - php: 5.6
       env:
-        - DB=MYSQL
+        - DB=PGSQL
         - PHPCS_TEST=1
         - PHPUNIT_TEST=framework
 
     - php: 7.0
       env:
         - DB=PGSQL
+        - PDO=1
```

Comment:
It'd probably be good to keep a PDO run on MySQL too

Reply:
one of the builds is7.2 / PDO / MySQL

Reply:
Ah sweet, I looked but didn't see